apps/examples/wifi_manager/wm_test/wm_test.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2022 Samsung Electronics All Rights Reserved.
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Do not add a commit to revert changes of your initial commits

Since initial commits are not merged, please make changes in the initial commit and re-push
Thereafter, discard these additional commits please.